native/src/components/MapView.tsx

```diff
@@ -101,8 +101,9 @@ const MapView = ({
   }
 
   const coordinates = selectedFeature?.geometry.coordinates
+  const defaultZoom = coordinates ? normalDetailZoom : defaultViewportConfig.zoom
   const defaultSettings: CameraSettings = {
-    zoomLevel: zoom ?? coordinates ? normalDetailZoom : defaultViewportConfig.zoom,
+    zoomLevel: zoom ?? defaultZoom,
```

Comment:
😍  I really like that you also took the time to refactor things just to make them easier to understand!